### ISSUE TITLE
bad request loading dashboards in 7.0

### ISSUE THREAD

**Issue by graphaelli** [elastic/apm-server#1997]
While dashboard loading has been deprecated since 6.4, `setup.dashboards.enabled=true` still triggers loading.  https://github.com/elastic/apm-server/issues/1905 worked around part of this by adding empty dashboards.  Unfortunately, this errors when apm-server tries to load the empty dashboards:

```
  error loading .../kibana/7/dashboard/apm-dashboards.json: returned 400 to import file: <nil>. Response: {"statusCode":400,"error":"Bad Request","message":"request body is required: [parse_exception] request body is required"}
```

This may unnecessarily trip people up when upgrading from 6.x to 7.0.

**Comment by graphaelli** [comment]
same as:
```
curl -i -XPOST -H 'kbn-version: 7.0.0-rc1' -H 'Content-Type: application/json' 'http://localhost:5601/api/kibana/dashboards/import?exclude=index-pattern&force=true' -d @_meta/kibana.generated/7/dashboard/apm-dashboards.json
```

**Comment by graphaelli** [comment]
closed by #1999
